Analysis/Smells/AstSmells/TooManyParametersArrowFunction.cs

```diff
@@ -0,0 +1,18 @@
+using Esprima.Ast;
+
+namespace BSDetector {
+    public class TooManyParametersArrowFunction : AstSmell {
+        public TooManyParametersArrowFunction() {
+            SmellName = "Too many parameters for arrow function";
+            SmellDescription = "Maximum recommended number of parameters for an arrow function is: 4.";
+        }
+        public override void AnalyzeNode(INode node, int depth) {
+            if (node is ArrowFunctionExpression) {
+                var ArrowFunctionNode = (ArrowFunctionExpression)node;
```

Comment:
If you use syntax like this, you won't need to use a cast in next line.
if (node is ArrowFunctionExpression arrowFunctionNode)